examples/dt/nova/nova-three-cells/README.md

```diff
@@ -0,0 +1,185 @@
+# Deployed Topology - Nova/MultipleCells
+
+Deploys an environment with three cells instead of the standard two cells
+
+## Purpose
+
+Focused on leveraging multiple nodesets to deploy two distinct cells 
+(cell1 and cell2) in addition to cell0. Computes will be allocated to both 
+cells
+
+## Environment
+
+### Nodes
+
+| Role              | Machine Type | Count |
+| ----------------- | ------------ | ----- |
+| Compact OpenShift | vm           |   3   |
+| OpenStack Compute | vm           |   4   |
+
+## Node topology
+TBD
+
+### Networks
+
+| Name         | Type     | Interface | CIDR            |
+| ------------ | -------- | --------- | --------------- |
+| Provisioning | untagged | nic1      | 172.22.0.0/24   |
+| Machine      | untagged | nic2      | 192.168.32.0/20 |
+| RH OSP       | trunk    | nic3      |                 |
+
+
+#### VLAN networks in RH OSP
+
+| Name        | Type        | CIDR              |
+| ----------- | ----------- | ----------------- |
+| ctlplane    | untagged    | 192.168.122.0/24  |
+| internalapi | VLAN tagged | 172.17.0.0/24     |
+| octavia     | VLAN tagged | 172.23.0.0/24     |
+| storage     | VLAN tagged | 172.18.0.0/24     |
+| storagemgmt | VLAN tagged | 172.20.0.0/24     |
+| tenant      | VLAN tagged | 172.19.0.0/24     |
+
+
+### Services, enabled features and configurations
+
+| Service          | configuration           | Lock-in coverage?  |
+| ---------------- | ----------------------- | ------------------ |
+| Barbican         | (default)               | Must have          |
+| Cinder           | LVM/iSCSI/lioadm        | Must have          |
+| Cinder Backup    | Swift                   | Must have          |
+| Glance           | Swift                   | Must have          |
+| Swift            | (default)               | Must have          |
+| Horizon          | N/A                     | Must have          |
+| Neutron          | Geneve (OVN)            | Must have          |
+| Swift            | (default)               | Must have          |
+
+#### Support services
+
+The following table lists services which are not the main focus of the testing
+(which may be covered by additional scenarios), but are required for the DT
+to work properly and can be deployed with any/default configuration.
+
+| Service          | Reason                     |
+| ---------------- |--------------------------- |
+| Keystone         | needed by all services     |
+
+
+### Additional configuration
+
+- Always-on, default services and features: TLSe
+- Two additional compute nodes
+
+#### Nova
+
+Increase default cells to include cell2
+
+```YAML
+---
+spec:
+  nova:
+    template:
+      secret: osp-secret
+      cellTemplates:
```

Comment:
please define the metadataServiceTemplate, either to run it globally, or per a cell
See https://github.com/bogdando/data-plane-adoption/blob/f61537f9701f6585183810b20e2142423ae4f8d0/docs_user/modules/proc_adopting-the-compute-service.adoc#L51 and https://github.com/bogdando/data-plane-adoption/blob/f61537f9701f6585183810b20e2142423ae4f8d0/docs_user/modules/proc_adopting-the-compute-service.adoc#L95 examples

If running it per a cell, we will also need a generated secret for neutron https://github.com/bogdando/data-plane-adoption/blob/f61537f9701f6585183810b20e2142423ae4f8d0/docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc#L285

Reply:
ack thanks, let me try and get a per cell working.

Reply:
please fill a bug - that is users facing manifest, it must contain explicit things, including metadataServiceTemplate (or providing it in kustomizations). However, ci-framework is NOT users facing, and doing implicit things there renders these manifests unusable outside of CI infra

Reply:
The question for me is the following: Does running kustomize on this DT locally will produce a complete CR? If so then I'm happy. If it does not produce complete CR but left out metadataTemplate definition then it is a bug indeed.